README.md

```diff
@@ -2,3 +2,74 @@ Fresque
 =======
 
 Fedora Review Server
+
+Installation
+----------------
+To download the project, open your terminal and type the following command
+
+```bash
+	$ git clone https://github.com/fedora-infra/fresque.git
+	$ cd fresque
+	$ python setup.py install
+```
+It will clone and install the project on your local machine
+
+Dependencies
+------------
+Fresque is a [Flask] application. The review data is stored into a relational database
+using [SQLAlchemy] as Object Relational Mapper and [alembic] to handle database
+schema changes.
+
+The dependecies can be found in the `requirements.txt` file of the project
+
+Running a development instance:
+-------------------------------
+First lets make a seperate virtual environment for the project
+to avoid conflicts of its dependecy with other python packages.
+after that install virtualenv wrapper which provides better CLI interface
+for the virtualenv.
+
+```bash
+	$ pip install virtualenv
+	$ pip install virtualwrapper
+```
+Then setup the path for the virtualenv by adding below lines
+in your `bashrc` file.
+
+```bash
+ 	export WORKON_HOME=$HOME/.virtualenvs
+  	export VIRTUALENVWRAPPER_VIRTUALENV_ARGS='--no-site-packages'
+  	export PIP_VIRTUALENV_BASE=$WORKON_HOME
+  	source /usr/local/bin/virtualenvwrapper.sh
+```
+
+Now clone the source:
+
+```bash
+	$ git clone https://github.com/fedora-infra/fresque.git
+	$ cd fresque
+	$ pip install -r requirements.txt
+	// set up the database before running the server
+	$ python createdb.py
+	// initiliase the alembic scripts in the fresque/lib folder
+	$ cd fresque/lib/
+	$ alembic init alembic
```

Comment:
@pypingou when I was trying to run the fresque first time after creating db. I was getting this [OSError] and so I run "alembic init alembic" to initialise the files.

[OSError]: https://gist.github.com/rahulrrixe/a4d2ee2627210320d07c